### ISSUE TITLE
scheduler will run out of connections

### ISSUE THREAD

**Issue by niusmallnan** [rancher/rancher#9306]
**Rancher versions:**
rancher/server: v1.6.3
scheduler: v0.8.1

**Steps to Reproduce:**
Get the scheduler container ID, maybe xxxxxx.
Expose the container namespace:
```
mkdir -p /var/run/netns/
CONTAINER_ID=xxxxxx
ln -sv /proc/$(docker inspect -f '{{.State.Pid}}' ${CONTAINER_ID})/ns/net /var/run/netns/${CONTAINER_ID}
```
**Results:**
The scheduler will run out of connections. Check with this cmd:
```
watch -n 2 "ip netns exec xxxxxx netstat -anp | grep ESTABLISHED "
```

**Comment by deniseschannon** [comment]
@niusmallnan `rancher/scheduler:v0.8.2` has been released. Can you make a new catalog entry into rancher-catalog in the `v1.6-development` branch? Please separate it into 2 commits (1 with a copy of the previous folder and a 2nd commit with the changes).

**Comment by niusmallnan** [comment]
@deniseschannon  Done

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/9332

**Comment by deniseschannon** [comment]
@niusmallnan Can you please not close the issues until QA has tested?

**Comment by deniseschannon** [comment]
Available with scheduler v0.8.2 in the `v1.6-development` branch.

**Comment by sangeethah** [comment]
Tested with rancher-server 1.6.3 and scheduler v0.8.2 in the v1.6-development branch.

Number of open connections held by scheduler is pretty low:
 
```
root@sangee163new-10acre-2:/home/sangeethahariharan1# ip netns exec 79806de4f3d4 netstat -anp | grep ESTABLISHED | wc -l
3
root@sangee163new-10acre-2:/home/sangeethahariharan1# 
```

With scheduler v0.8.2 , number of open connections keeps increasing.
```
root@sangee163new1-10acre-1:/home/sangeethahariharan1# ip netns exec 0b90f5963f1b netstat -anp | grep ESTABLISHED | wc -l
3980
```
